java-frontend/src/main/java/org/sonar/java/se/FlowComputation.java

```diff
@@ -191,33 +193,42 @@ private static boolean isMethodInvocationNode(ExplodedGraph.Node node) {
     return false;
   }
 
-  private static int correspondingArgumentIndex(SymbolicValue candidate, ExplodedGraph.Node invocationNode) {
+  private static int correspondingArgumentIndex(Set<SymbolicValue> candidates, ExplodedGraph.Node invocationNode) {
     MethodInvocationTree mit = (MethodInvocationTree) invocationNode.programPoint.syntaxTree();
     List<SymbolicValue> arguments = argumentsUsedForMethodInvocation(invocationNode, mit);
-    return arguments.indexOf(candidate);
+    // FIXME whatif multiple candidates match some of the arguments?
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1134)